pkg/agent/testdata/AKSUbuntu1804+Containerd+IPMasqAgent/line48.sh

```diff
@@ -153,22 +153,26 @@ installStandaloneContainerd() {
     # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
     # v1.4.1 is our lowest supported version of containerd
-    local CONTAINERD_VERSION="1.4.4"
+    local CONTAINERD_VERSION="1.5.0-beta.git31a0f92df"
     if semverCompare ${CURRENT_VERSION:-"0.0.0"} ${CONTAINERD_VERSION}; then
         echo "currently installed containerd version ${CURRENT_VERSION} is greater than (or equal to) target base version ${CONTAINERD_VERSION}. skipping installStandaloneContainerd."
     else
         echo "installing containerd version ${CONTAINERD_VERSION}"
         removeMoby
         removeContainerd
         updateAptWithMicrosoftPkg
-        apt_get_install 20 30 120 moby-containerd=${CONTAINERD_VERSION}* --allow-downgrades || exit $ERR_CONTAINERD_INSTALL_TIMEOUT
+        # TODO: first try downloading from microsoft apt repo then fallback to our storage ep
```

Comment:
not ideal that for this build we have to revert back to using the storage ep.
maybe we should consider always tryi to apt-get first and then fail fast if it can't find the specified version on the apt repo and try downloading it from storage 
with this + having rp being able to specify version overrides hopefully it'll be easier to quickly onboard new versions

Reply:
or make it simple that we do apt-get install for 1.x.y and install from by for 1.x.y-beta-*?